prow/cmd/clonerefs/main.go

```diff
@@ -111,6 +125,11 @@ func main() {
 		if jobRefs.Type != kube.PeriodicJob {
 			// periodic jobs do not configure a set
 			// of refs to clone, so we ignore them
+			for _, gitRef := range o.refs.gitRefs {
+				if gitRef.Org == jobRefs.Refs.Org && gitRef.Repo == jobRefs.Refs.Repo {
+					logrus.Fatalf("Clone specification for %s/%s found both in Prow variables and user-provided flags", jobRefs.Refs.Org, jobRefs.Refs.Repo)
```

Comment:
Why is this a fatal? It breaks our image-registry [tests](https://ci.openshift.redhat.com/jenkins/job/test_pull_request_image_registry_extended/235/consoleFull#-56402893958b6e51eb7608a5981914356).

Reply:
The source of truth should be the environment variable, right? We will likely need to update our image-registry jobs to not clone image-registry?

Reply:
hmm, perhaps this should simply log an error and prioritize the env instead @stevekuznetsov

Reply:
@BenTheElder @stevekuznetsov Guys, please unblock us!